ext/tests/return_expanded_params_internal.phpt

```diff
@@ -2,6 +2,8 @@
 Check if pre hook can expand and then return $params of internal function
 --DESCRIPTION--
 The existence of a post callback is part of the failure preconditions.
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 80200) die('skip requires PHP >= 8.2'); ?>
```

Comment:
I think this should work in 8.1?

Reply:
Same thing like `expand_params_internal.phpt`